api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -197,6 +190,17 @@ message JwtProvider {
   //       exp: 1501281058
   //
   string payload_in_metadata = 9;
+
+  // The two below fields define the amount of slack in seconds that will be used
+  // when determining if a JWT is valid or has expired. Validity is determined by
+  // the formula: VALID_FROM("iat") + nbf_slack <= NOW <= VALID_TO("exp") + exp_slack
+  // which aims to provide a remedy in the following case:
+  // Some load balancer performinc OIDC authentication receives a request and determines that the 
+  // existing access token is valid (for another 1 second). It forwards the request to Istio Ingressgateway 
+  // and subsequently to some pod with an envoy sidecar. Meanwhile, 1 second has passed and when envoy checks 
+  // the token it finds that it has expired.   
+  uint32 nbf_slack = 0;
```

Comment:
In protobuf:  it is

type  field_name = field_index;

field_index has to be unique for all fields in a message. 

There is not way to default default value in protobuf.  default value is determined by the type